arroyo/processing/strategies/batching.py

```diff
@@ -143,15 +143,6 @@ def submit(self, message: Message[TPayload]) -> None:
 
         start = time.time()
 
-        self.__metrics.timing(
-            "receive_latency",
-            (start - message.timestamp.timestamp()) * 1000,
-            tags={
-                "topic": message.partition.topic.name,
-                "partition": str(message.partition.index),
-            },
-        )
```

Comment:
What happened to this ?

Reply:
A message here is no longer necessarily a BrokerPayload with a singular partition, topic, offset.

Reply:
I doubt it, unless you want to keep it for Snuba's replacer. Put it back anyway.